pkg/envoy/agent.go

```diff
@@ -149,6 +149,11 @@ func (a *Agent) terminate() {
 		ticker := time.NewTicker(activeConnectionCheckDelay)
 		for range ticker.C {
 			ac := a.activeProxyConnections()
```

Comment:
did you check it will still show up when zero? since usedonly was set I wasn't 100% sure

Reply:
Yes.  That is correct. But my thought process was if downstream_cx_active is not there at all in the stats, we can skip active connections. May be we should reword the error message?